src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5032,4 +5032,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_InvalidPreprocessingSymbol" xml:space="preserve">
     <value>Invalid name for a preprocessing symbol; '{0}' is not a valid identifier</value>
   </data>
+  <data name="ERR_FeatureNotAvailableInVersion7_1" xml:space="preserve">
+    <value>Feature '{0}' is not available in C# 7.1.  Please use language version {1} or greater.</value>
```

Comment:
Paste issue from similar messages :) will fix all of them now.

---
In reply to: [106479940](https://github.com/dotnet/roslyn/pull/17894#discussion_r106479940) [](ancestors = 106479940)